app/scrapers/youtube.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Youtube(Scraper):
+    """Scraper class for Youtube"""
+
+    def __init__(self):
+        self.url = 'https://www.youtube.com/results'
+        self.queryKey = 'search_query'
+
+    def parseResponse(self, soup):
+        """ Parse the response and return set of urls
```

Comment:
This is not a __set of URLs__.  It is a __list of dicts__.  A set guarantees not repeats and this code does not do that.  The items in the list are dicts that contain titles and links (urls).

[[Tile1,url1], [Title2, url2],..] should be changed to [{'title': Tile1, 'link': url1}, {'title': Tile2, 'link': url2}, ...] to make the result more clear to the reader